dotnet/src/VectorDataIntegrationTests/SqliteVecIntegrationTests/Support/SqliteTestStore.cs

```diff
@@ -26,7 +26,7 @@ private SqliteTestStore()
     protected override Task StartAsync()
     {
         this._databasePath = Path.GetTempFileName();
-        this._connectionString = $"Data Source={this._databasePath}";
+        this._connectionString = $"Data Source={this._databasePath};Pooling=false";
```

Comment:
The cleanup removes the file and it was sometimes failing with file in use. Disabling the pooling solves the problem (as there are no open connections in the pool I guess)